docs/source/interfaces/PyChop.rst

```diff
@@ -0,0 +1,191 @@
+calculate the resolution for as an input, can be directly passed to third party 
+programs for resolution convolution purposes.
+
+For further help, use ``help(PyChop2)`` after importing the class.
+
+Theory
+------
+
+The energy resolution calculated by ``PyChop`` has contributions from the time
+width of the moderator pulse :math:`\tau_{\mathrm{mod}}`, the opening times of the 
+choppers, :math:`\tau_{\mathrm{chop}}`, the response time of the detector,
+:math:`\tau_{\mathrm{det}}`, and the effect of the sample, :math:`\tau_{\mathrm{sam}}`.
+The first two contributions dominate so we will only concentrate on those.
+
+The moderator time width is determined from fitting data above 100 meV to a 
+:math:`\chi^2` distribution `[1]`_ which has a variance :math:`\tau_{\mathrm{mod}}^2
+=3/(\Sigma v)^2` where :math:`\Sigma` is the macroscopic scattering cross-section
+of the moderator and :math:`v` is the neutron velocity. However, experimentally
+it was found that this underestimates the widths at high energy `[2]`_, so that a 
+modified form for the variance
+
+.. math:: \tau_{\mathrm{mod}}^2 = \tau_0 + \frac{3}{(\Sigma v)^2}
+
+is used in PyChop. In future versions, the moderator lineshape will be reparameterised
+to use an Ikeda-Carpenter lineshape, which more accurately describes the ToF spectrum
+at lower neutron energies.
+
+The chopper time width is determined from the geometry of chopper and is given by
+`[2]`_, `[3]`_
+
+.. math:: 
+        \tau_{\mathrm{chop}}^2 \left\{ \begin{array}{ll} \frac{(\Delta T)^2}{6} 
+        \left[\frac{1-\gamma^4/10}{1-\gamma^2/6}\right] & 0 \leq \gamma < 1 \\
+        \frac{(\Delta T)^2}{6} \left[\frac{3}{5} 
+        \frac{\gamma(\sqrt{\gamma}-2)^2(\sqrt{\gamma}+8)}{\sqrt{\gamma}+4}\right]
+        & 1 \leq \gamma < 4 \\
+        \mathrm{undefined} & \gamma \geq 4 \end{array} \right.
+
+where
+
+.. math:: \begin{array}{rcl} \Delta T &=& \frac{p}{2R\omega} \\
+        \gamma &=& \frac{2R}{\Delta T} \left| \frac{1}{s} - \frac{1}{v} \right| \\
+        s &=& 2\omega\rho \end{array}
+
+and :math:`p` is the width of the slits of the Fermi chopper, :math:`R` is the radius
+of the chopper package (assumed cylindrical), :math:`\omega` is its rotation speed, 
+:math:`v` is the neutron velocity and :math:`\rho` is the curvature of Fermi chopper 
+slits.
+
+The time variances above are defined at the moderator and chopper positions respectively.
+As the neutron bunches travel towards the sample and detector they also spread out,
+and the final time (energy) widths are determined by the geometry (distances) of the
+instrument. Specifically, the relative energy width is given by the sum in quadrature
+of each of the contributing time widths, which we will restrict here to the two major
+terms, :math:`\tau_{\mathrm{mod}}` and :math:`\tau_{\mathrm{chop}}` `[4]`_:
+
+.. math:: \left( \frac{\Delta E}{E_i}\right )^2 = 
+        \left[ 2\frac{\tau_{\mathrm{chop}}}{t_{\mathrm{chop}}} \frac{l_0+l_1}{l_2}
```

Comment:
There should be `1+\frac{l_0+l_1}{l_2} \left(\frac{E_f}{E_i}\right)^\frac{3}{2}`